01-ESP32_firmware/include/MQSensor.hpp

```diff
@@ -6,39 +6,43 @@
 /**
  * MQ-7 and MQ-135 sensors driver class.
  */
-class MQSensor
+class MQSensor ///< MQSensor class \date $Date: 2020/05/20 14:16:20 $
 {
 private:
-    uint8_t analogPin;
-    uint8_t enablePin;
+    uint8_t analogPin;  
```

Comment:
Unikaj committowania takich zmian. Nawet jak dodasz spację, to git automatycznie uważa, że zmieniłeś całą linię kodu i robi się syf w blejmach.